test/helpers/loanTestHelpers.js

```diff
@@ -374,13 +374,13 @@ async function calcLoanValues(rates, product, collateralWei) {
     ret.repaymentAmount = ret.tokenValue
         .mul(product.collateralRatio)
         .div(ppmDiv)
-        .round(0, BigNumber.ROUND_HALF_UP);
+        .round(0, BigNumber.ROUND_DOWN);
```

Comment:
i don't see the motivation for this change. if the math works out with `ROUND_HALF_UP` why not just keep that?

Reply:
The ultimate test case to demonstrate the requirement is on the UI (see cypress test)
The aim is to have the exact actual loanAmount, repaymentAmount  calculated and shown for the user on the new loan form. 
It was very complex to achieve that with rounded divs on UI (actually I gave it up). And for loans we don't need rounded divs (unlike with fill on exchange). 
Open for change but then the UI calc has to be updated (see `newLoanForm.js` on 'loans_refactor' branch augmint-web )